### DISCUSSION TITLE
Debug under 202303

### DISCUSSION THREAD

**Discussion by plotn** [dbeaver/dbeaver#19626] [DBeaver Development]
### Description

I'm trying to compile uder 202303 (is it possible due to: https://github.com/dbeaver/dbeaver/pull/19524 ?)
Do everything as described in: https://dbeaver.com/docs/wiki/Develop-in-Eclipse/

"mvn clean install" works fine but debug fails:

![image](https://user-images.githubusercontent.com/11477081/230407976-0b4693cd-a748-4248-a322-dccdc0d70512.png)

what I did wrong?

### DBeaver Version

current devel

### Operating System

win 10 x64

**Comment by LonwoLonwo** [comment]
You can add all dependencies manually:

![2023-04-06 17_39_52-Debug Configurations](https://user-images.githubusercontent.com/45152336/230413640-b39601f6-5b2a-4514-af0b-05b5d6d0d646.png)

**Comment by plotn** [comment]
but i havent javax.xml:

![image](https://user-images.githubusercontent.com/11477081/230415617-d2ccc878-1b1a-472e-aad5-b6b9703c646f.png)

**Comment by plotn** [comment]
Still got an errors:
![image](https://user-images.githubusercontent.com/11477081/230896653-d98e3f38-c121-4fda-8de4-58193155b65b.png)

then:
![image](https://user-images.githubusercontent.com/11477081/230896708-03064eec-a7b7-4afe-93a3-4ff0f35bd543.png)

then it runs, but I got, fore example, very strange setting window:

![image](https://user-images.githubusercontent.com/11477081/230896840-c2928090-b3af-4e8f-aa58-270d2e355742.png)

(as like from "common" eclipse, not from dbeaver)

**Reply by E1izabeth** [reply]
Probably, some non-dbeaver plugins were checked

**Comment by plotn** [comment]
So I changed 
Fragment-Host: org.slf4j.api
to
Fragment-Host: slf4j.api

in a couple of files and everything looks good - it compiles, starts.
A rest of errors:

![image](https://user-images.githubusercontent.com/11477081/230901704-c25b8fbc-1569-4389-a942-0c061eb6e7ab.png)

But the settings still looks weird

**Reply by E1izabeth** [reply]
If you don't have some libraries, you could try downloading them from orbit repository https://download.eclipse.org/tools/orbit/downloads/drops/R20221123021534/repository/ via `Install new software` eclipse dialog
